boards/OAK-D-CM4-POE-C11.json

```diff
@@ -0,0 +1,54 @@
+{
+    "board_config":
+    {
+        "name": "OAK-D-CM4-C11",
+        "revision": "R2M0E2",
+        "cameras":{
+            "CAM_A": {
+                "name": "rgb",
+                "hfov": 80.0,
```

Comment:
OAK-D-CM4-POE-C11 (w/OV9782) HFOV is the same as stereo HFOV so I suppose 71.86°